docs/arcaflow/getting-started.md

```diff
@@ -1,101 +1,97 @@
-#!/usr/local/bin/python3
-import dataclasses
-import sys
-from arcaflow_plugin_sdk import plugin
-
-
-@dataclasses.dataclass
-class InputParams:
-    name: str
-    
-    
-@dataclasses.dataclass
-class SuccessOutput:
-    message: str
+```bash
+git clone https://github.com/arcalot/arcaflow-workflows.git
+export WFDIR=arcaflow-workflows/basic-examples/basic/
+```
 
+Then we will run the workflow, setting the working directory as the context, and defining the workflow, configuration, and input files to use:
 
-@plugin.step(
-    id="hello-world",
-    name="Hello world!",
-    description="Says hello :)",
-    outputs={"success": SuccessOutput},
-)
-def hello_world(params: InputParams):
-    return "success", SuccessOutput(f"Hello, {params.name}")
+```bash
+arcaflow --context $WFDIR --workflow workflow.yaml --config config.yaml --input input.yaml
+```
 
+Arcaflow will display logs, depending upon the configured verbosity, and then will return the machine-readable output of the workflow in YAML format:
 
-if __name__ == "__main__":
-    sys.exit(
-        plugin.run(
-            plugin.build_schema(
-                hello_world,
-            )
-        )
-    )
+```bash
+output_data:
+    example:
+        message: Hello, Arcalot!
+output_id: success
 ```
 
-!!! tip
-    Further reading: [Creating your first Python plugin](plugins/python/first.md)
-
-## Step 4: Build the plugin
+[Learn more about running workflows &raquo;](/arcaflow/running/){ .md-button }
 
-Next, let's create a `Dockerfile` and build a container image:
+## Writing Workflows
+As a workflow author, you determine the steps of the workflow, how data will pass between the steps, what input is required from the workflow user, and what output will be returned. It is possible to build very complex workflows with data translations, sub-workflows, parallelisim and serialization, and multiple output paths.
 
-```Dockerfile title="Dockerfile"
-FROM python:alpine
+Let's start with something simple. Our workflow will collect a `nickname` input from the user and will pass that input to an example "Hello world!" step. The workflow will also run a UUID generateion step in parallel to the example step, and it will return both the UUID and a greeting.
 
-ADD plugin.py /
-RUN chmod +x /plugin.py && pip install arcaflow_plugin_sdk
+In the first part of the `workflow.yaml` file we will define the workflow compatibility version and the input schema for the workflow. In the input schema, we are expecting only a single input called `nickname` with a type of `string`.
 
-ENTRYPOINT ["/plugin.py"]
-CMD []
+```yaml title="workflow.yaml (excerpt)"
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        nickname:   <<== Input key name
+          display:
+            description: Just a name
+            name: Name
+          required: true
+          type:
+            type_id: string   <<== Input value type
+...
```

Comment:
It may make sense in this example, as well as the others, to just put a comment character before the comments you added. I can imagine a fraction of users to think that actually is supposed to be in the YAML.